docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
+Bootstrapping your local node is easy. Once you have an active node running, bootstrap it with:
 
 ```shell
 dune --bootstrap-system-full
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts. 
 
-To create a new account:
+The following command creates a new account:
 
 ```shell
 dune --create-account <ACCOUNT_NAME>
 ```
 
-To get account info:
+The following command gets account info:
 
 ```shell
 dune -- cleos get account <ACCOUNT_NAME>
 ```
+## Smart contract development
 
-## Smart Contract Development
+Create a sample project to learn how to use DUNE to compile, deploy, and interact with smart contracts.
 
-Let's create a sample project so that we can learn how to compile, deploy, and interact with smart contracts using DUNE.
+### Create sample project
 
-Navigate to a directory you want to create a project in, and then run the following command:
+Navigate to a directory in which you want to create a project, then run the following command:
 
 ```shell
 dune --create-cmake-app hello .
 ```
-
-This will create a `hello` directory with a cmake style EOS smart contract project.
+This command creates a `hello` directory with a cmake style EOS smart contract project.
 
 Replace the contents of `src/hello.cpp` with the following code:
 
 ```cpp
-#include <eosio/eosio.hpp>
+#include <eosio/eosio.hpp> 
 using namespace eosio;
 
-CONTRACT hello : public contract {
-   public:
-      using contract::contract;
+CONTRACT hello : public contract {ompilinc
+ public:
+ using contract::contract;
 
       TABLE user_record {
          name user;
-         uint64_t primary_key() const { return user.value; }
+ uint64_t primary_key() const { return user.value; }
       };
-      typedef eosio::multi_index< name("users"), user_record> user_index;
+ typedef eosio::multi_index< name("users"), user_record> user_index;
 
       ACTION test( name user ) {
-         print_f("Hello World from %!\n", user);
+ print_f("Hello World from %!\n", user);
          user_index users( get_self(), get_self().value );
-         users.emplace( get_self(), [&]( auto& new_record ) {
-            new_record.user = user;
+ users.emplace( get_self(), [&]( auto& new_record ) {
+ new_record.user = user;
          });
       }
 };
 ```
-
 ### Compile the contract
 
 From the root of your project, run the following command to compile your contract:
 
 ```shell
 dune --cmake-build .
 ```
-You will see your contract being compiled. If there are any errors you will see them in the output.
+You see your contract compiling. If there are any errors, you see them in the output.
 
 ### Deploy your contract
 
-We need to create an account for your contract, and then we can deploy it.
+Use the following commands to create an account and deploy the contract to the account.
```

Comment:
First-person plural, or second-person. 

This sterilizes the content and makes it commands instead of a journey with the instructor.

Reply:
This construction is second-person point of view. It is called "implied". "Implied subjects occur when a sentence does not state the doer of the action, but it is clear to whom the sentence is referring." This construction is a standard practice for instructions.

Reply:
The journey includes entering commands. Plus it is not done by more than one person (the reader). It's not good practice to imply others are involved with the action without explicitly identifying them.